tests/pulseaudio_dbus_spec.lua

```diff
@@ -202,4 +203,19 @@ b.describe("PulseAudio with DBus", function ()
 			       end
 			   end
 		   end)
+
+		   b.it("Will Cycle through all available PlaybackStreams and move them to the FallbackSink", function()
```

Comment:
This seems to be quite a complicated test to run.
We'd have to have a couple of streams playing to verify the behavior.

I think that it would be much more useful if:

1. we moved it in its own `describe` block
2. we added some mechanism to start a couple of streams in the background (with the audio muted)
3. we added some assertions to make this an actual test (see comment below)